### PR TITLE
Support for disabled sails globals

### PR DESCRIPTION
**Pull request by jifeon**
It's a PR for #12

### CODE REVIEW

File: api/services/passport.js [hunk 1]
```diff
@@ -63,7 +64,6 @@ passport.protocols = require('./protocols');
  * @param {Function} next
  */
 passport.connect = function (req, query, profile, next) {
-  var strategies = sails.config.passport;
```
Comment (jifeon) [inline api/services/passport.js]: It was unused..

**Comment by jifeon** [comment]
I checked it only with my project with disabled globals. Authorization through github works great. I did not test all cases but changes are minor and shouldn't break anything. Btw, looks like tests are empty, am I right?

**Comment by tjwebb** [comment]
I think you need to merge in the last code into your fork. There are unit tests

**Comment by jifeon** [comment]
Rebased, tests pass without errors :)

**Comment by tjwebb** [comment]
Sweet, thanks.

**Pull request merged**